google-cloud-spanner/src/main/java/com/google/cloud/spanner/Spanner.java

```diff
@@ -16,10 +16,13 @@
 
 package com.google.cloud.spanner;
 
-import com.google.api.core.ApiFuture;
 import com.google.cloud.Service;
 
-/** An interface for Cloud Spanner. */
+/**
+ * An interface for Cloud Spanner. Typically, there would only be one instance of this for the
+ * lifetime of the application which must be closed by invoking {@link #close()} when it is no
+ * longer needed.
```

Comment:
+ something like "Failure to do so may result in leaking session resources and exhausting session quota."